core/domain/skill_domain.py

```diff
@@ -416,20 +416,22 @@ def to_dict(self):
         }
 
     @classmethod
-    def create_default_skill(cls, skill_id):
+    def create_default_skill(
+            cls, skill_id, description=feconf.DEFAULT_SKILL_DESCRIPTION):
```

Comment:
Should pass the description in explicitly, don't need a default. Let's also remove feconf.DEFAULT_SKILL_DESCRIPTION altogether.

Reply:
Done.